docs_src/examples/Ch-ExamplesDataandRetention.md

```diff
@@ -0,0 +1,44 @@
+# Data Retention and Persistent Caps
+
+!!! note
```

Comment:
PLease a `edgey` comment that this is new in EdgeX 3.1

Reply:
updated